src/ng/rootScope.js

```diff
@@ -907,8 +911,7 @@ function $RootScopeProvider() {
         this.$$destroyed = true;
 
         if (this === $rootScope) {
-          //Remove handlers attached to window when $rootScope is removed
-          $browser.$$applicationDestroyed();
+          $shutdown();
```

Comment:
I am not sure we need this, now that `$shutdown` is a public service.

Reply:
Well, it's been a while, but what I think I meant that this was a poor man's `$shutdown()`. Before `$shutdown()` was implemented, we were special-casing `$rootScope.$destroy()` to do destroy the application (because `$browser.$$applicationDestroyed()` was private). But now that `$shutdown()` is a public service, people wanting to destroy the app can (and should) use that (instead of destroying `$rootScope`.

It doesn't seem too harmful though, so we might as well keep it to avoid the breaking change.